api/capi/include/nnstreamer-single.h

```diff
@@ -185,6 +185,33 @@ int ml_single_set_input_info (ml_single_h single, const ml_tensors_info_h info);
  */
 int ml_single_set_timeout (ml_single_h single, unsigned int timeout);
 
+/**
+ * @brief Sets the property value for the given model.
```

Comment:
How about adding a note that the behavior of this function changes before and after an invoke?
Before the first invoke call, the properties can be updated freely as the framework is not yet open.
However, after the first invoke (ml_single_invoke), the support of updating the properties will be limited depending on the support provided by the framework.